lib/might/filter_parameters_extractor.rb

```diff
@@ -1,6 +1,7 @@
 require_relative 'filter_predicates'
 require_relative 'filter_undefined_parameter'
 require_relative 'filter_parameter'
+require_relative 'filter_parameters'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.